### PR TITLE
Deprecated alias on DI Extension

### PR DESCRIPTION
**Pull request by maxhelias**
Fix issue #1067

### CODE REVIEW

**Comment by maxhelias** [comment]
@stof Can you tell me if it's ok like that ?

File: DependencyInjection/DoctrineExtension.php [hunk 1]
```diff
@@ -92,6 +92,12 @@ protected function dbalLoad(array $config, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('dbal.xml');
 
+        // Deprecated
```
Comment (alcaeus) [inline DependencyInjection/DoctrineExtension.php]: The comment isn't necessary ;)
```suggestion
```

File: DependencyInjection/DoctrineExtension.php [hunk 2]
```diff
@@ -348,6 +354,11 @@ protected function ormLoad(array $config, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
         $loader->load('orm.xml');
 
+        // Deprecated
```
Comment (alcaeus) [inline DependencyInjection/DoctrineExtension.php]: The comment isn't necessary ;)
```suggestion
```

**Comment by alcaeus** [comment]
Thanks @maxhelias!

**Comment by stof** [comment]
@alcaeus I suggest making a release for that, to avoid getting lots of bug reports for that from people using stable releases. What do you think ?

**Comment by alcaeus** [comment]
Done.

**Pull request merged**